Failed to find Electron v7.0.0 for linux-armv7l

When running `npm install --save-dev electron-mksnapshot` on my raspbery pi 4, I get this result:
```
> electron-mksnapshot@7.0.0 install /home/pi/Developer/atom/node_modules/electron-mksnapshot
> node ./download-mksnapshot.js

/home/pi/Developer/atom/node_modules/electron-mksnapshot/download-mksnapshot.js:19
  if (err != null) throw err
                   ^

Error: Failed to find Electron v7.0.0 for linux-armv7l at https://github.com/electron/electron/releases/download/v7.0.0/mksnapshot-v7.0.0-linux-armv7l.zip
    at Request.<anonymous> (/home/pi/Developer/atom/node_modules/nugget/index.js:169:61)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at Request.onRequestResponse (/home/pi/Developer/atom/node_modules/request/request.js:1066:10)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:552:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:119:17)
    at TLSSocket.socketOnData (_http_client.js:454:20)
    at emitOne (events.js:116:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electron-mksnapshot@7.0.0 install: `node ./download-mksnapshot.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron-mksnapshot@7.0.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2019-10-29T19_35_39_121Z-debug.log
```

The log can be found here: https://hastebin.com/igonihavoh.log

I have run `npm install electron --save-dev` which told me it had installed electron.

--- a/download-mksnapshot.js
+++ b/download-mksnapshot.js
@@ -3,13 +3,16 @@ var path = require('path')
 var electronDownload = require('electron-download')
 var extractZip = require('extract-zip')
 var versionToDownload = require('./package').version
-
+var archToDownload = process.env.npm_config_arch
+if (archToDownload === 'arm64') {
+  archToDownload = 'arm64-x64'
+}
 function download (version, callback) {
   electronDownload({
     version: version,
     mksnapshot: true,
     platform: process.env.npm_config_platform,
-    arch: process.env.npm_config_arch,
+    arch: archToDownload,
     strictSSL: process.env.npm_config_strict_ssl === 'true',
     quiet: ['info', 'verbose', 'silly', 'http'].indexOf(process.env.npm_config_loglevel) === -1
   }, callback)

